portal/v2/src/react-app-env.d.ts

```diff
@@ -0,0 +1 @@
+/// <reference types="react-scripts" />
```

Comment:
do we need this file?

Reply:
This script seems to get generated from the react-scripts npm library. So we could just leave it as it is or potentially remove it and perform a git ignore on it?